fastlane/lanes/localization.rb

```diff
@@ -4,10 +4,10 @@
 # Constants
 #################################################
 
-# URL of the GlotPress project which contains App strings
+# URL of the GlotPress project containing the app's strings
```

Comment:
This makes the wording consistent with the other comment just below this.

I went for using containing here, instead of updating the other one to use "which contains" because this option seems smoother to me.